nfadeeva/assignment2/Assignment_2_Q1.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+
+
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.right = None
+        self.left = None
+
+
+def find_all_ancestors(root, key, result=None):
+        """
+        Prints all the ancestors of the given key
+
+        :return list of ancestors of the given key or False if the key is not in the Tree
+        """
+
+        if root is None:
+            return False
+
+        if result is None:
+            result = []
+
+        if root.data == key:
```

Comment:
Can you return [root.data] here for consistency?

Reply:
Fixed it